lib/Sub/Util.pm

```diff
@@ -21,6 +21,28 @@ $VERSION   = eval $VERSION;
 require List::Util; # as it has the XS
 List::Util->VERSION( $VERSION ); # Ensure we got the right XS version (RT#100863)
 
+use constant CAN_SUB => $^V >= 5.016;
```

Comment:
Should check against $], not $^V.

Reply:
`$^V` is not available in 5.6 for example and stands for `v5.x` in later version
even if the compare is not incorrect for modern Perl version
you should use `$]`